package.json

```diff
@@ -2,13 +2,17 @@
   "name": "@zenika/sensei",
   "version": "1.0.0",
   "scripts": {
+    "postinstall": "npm install --prefix=src/pdf",
```

Comment:
I'm not in favor of this change. I separated the pdf package because I did not want the default install to include the downloading of Chrome. I'd like to keep it that way because I think it makes sense. Couldn't we make the CLI run `npm install` in the `pdf` folder only when `sensei pdf` is run? I think that would a better.

Reply:
I don't think it's a good idea to install dependency when using tool. PDF generation is already a slow process so if we add delay for download dependencies so experience usage will be deteriorate :/

Reply:
If you really want to separate pdf of the other usage then maybe we also should separate CLI ? One for pdf and an other for serve

Reply:
I think it's reasonable and here's why. Yes, PDF generation is slow, so the install time is negligible and only happens the first time. Compare with the use case of someone who only wants to serve the slide and who has to wait for Chrome to install for no reason.

That said, if we intend to distribute sensei as a CLI that users only install/update once in a while, then I guess it makes sense for everything (including the big Chrome download) to happen at install time. In that new context, the PDF package does not need to be separated and all dependencies and scripts could be lifted to the root package.

What do you think?

Reply:
Just giving my humble opinion: I want to use sensei as a container. I think that having only one cli simplify the use of it but I will still use it as a container. So the `npm i` will be executed at the container image build. Not sure it helps 😅

Reply:
I agree with @ebriand , as a user I prefer to use only one cli to simplify usage. Docker image must be provided to only wrap CLI for those who don't want to install nodejs on their device.

Actually their is a real benefit for provide a versioned training material with sensei tool (with package.json dependency) but I'll personally use tool installed as global or docker image thus compatibility issues could still occurred. 

So I will suggest to not install sensei as training dependency but as global tool CLI. The huge benefit is to simplify training repo to contains only slides & labs.

Maybe you could create a poll to get option for a large trainer panel ?

Reply:
No, I don't want to spend time waiting for a poll result. I think this route is completely reasonable and we should take it.

Reply:
It's completely understandable ! So I'm not sure to know what next should be done to close this PR ?

Reply:
Agree with you !